api/base/parsers.py

```diff
@@ -137,6 +137,20 @@ def parse(self, stream, media_type=None, parser_context=None):
         else:
             raise JSONAPIException(source={'pointer': '/data'}, detail=NO_DATA_ERROR)
 
+    def flatten_multiple_relationships(self, parser, relationships):
+        rel = {}
+        for resource in relationships:
+            ret = super(parser, self).flatten_relationships({resource: relationships[resource]})
```

Comment:
Didn't dig into this long, I'm sure there's a better way to do this then passing in the parser from the child class.  

I consolidated `flatten_multiple_relationships` here to be called on two child parsers.  I wanted to call `flatten_relationships` method defined on this parent class but it's being called from the child class.

My main goal was to prevent this situation from occurring again - the two parsers diverging in behavior.